app/components/auth/NoUserAvatar.js

```diff
@@ -0,0 +1,22 @@
+import { FaUser } from "react-icons/fa";
+
+export function NoUserAvatar({size,name}){
+    const char = name ? name.charAt(0) : null;
+    return (
+        <div 
+            className="d-flex justify-content-center align-items-center"
+            style={{
+                width: `${size}px`, 
```

Comment:
I added size variable because I wanted to use this component at two places but with different size. One in the menubar and the other when the profile icon is clicked to open user info diaglog.